### DISCUSSION TITLE
Has anyone experienced CPU consumption issue?

### DISCUSSION THREAD

**Discussion by RoadToDream** [BartoszCichecki/LenovoLegionToolkit#902] [Q&A]
Not sure when does this happen on my end, but it seems while the LLT is running in background, it constantly takes a 3-5% of CPU usage. 

![image](https://github.com/BartoszCichecki/LenovoLegionToolkit/assets/8088354/9a2ae404-ab2c-4722-8f57-00529cf7a0f3)

**Comment by BartoszCichecki** [comment]
I don't know why, but there are some features that require more CPU like: Actions based on process name (App started/stopped) and Aurora keyboard backlight. Around 3-4% CPU usage is normal if Aurora is set as keyboard effect.

**Reply by RoadToDream** [reply]
I fully understand that. Issue is I haven't enabled any of those features...

**Comment by tonydlo** [comment]
I'm also noticing background activity even though I'm using no actions or keyboard lighting. (Basically stock settings.)

I'm on the new Legion Slim 5 16APH8 Gen 8 using Toolkit 2.15.4 (latest as of this writing).

It's easiest to see this in Modern Standby mode on battery. It is the highest drain on the battery.

![image](https://github.com/BartoszCichecki/LenovoLegionToolkit/assets/13370429/0631a8da-3986-4c08-8ac7-d57b2f3788eb)

**Reply by BartoszCichecki** [reply]
I wouldn't pay too much attention to these statistics, Windows isn't great a keeping track of it. There is bg activity, LLT is listening for events sent by EC all the time so that isn't surprising.

**Reply by tonydlo** [reply]
Good to know, thanks!

**Reply by ghost** [reply]
Any plans to have a setting to reduce or even stop the EC calls? Noticing that LLT seems to take a good small chunk of power with it as it runs in the background.

**Reply by BartoszCichecki** [reply]
You are welcome to suggest changes in a PR.
